source/systems/happinessandsaturation.md

```diff
@@ -10,17 +10,21 @@ The citizens, however, will NOT die if their saturation goes down to 0, they wil
 
 ## Happiness System
 
-There is an **overall** colony happiness and an **individual** worker happiness.
+There is an **overall** colony happiness and an **individual** citizen happiness. Higher colony happiness increases the initial level and skills of new colonists. Higher citizen happiness increases the rate the colonist's gains Intelligence when studying at a [Library](../../source/buildings/library).
 
 Overall colony happiness is 1-10 (initially set to 5). Happiness depends on three basic factors: **security, housing, and saturation**.
 
-Every night the overall colony happiness is recalculated based on the following:
+If a colonist's saturation above 7 shanks, they will be more happy.
+If the colonist's home (or for [guards](../../source/workers/guard), or [trainees](../../source/workers/archerintraining), workplace) is above level 2, the colonist will be more happy.
+If two [Guards](../../source/workers/guard) are available for every three Workers, the colonists will be more happy.
 
-- If the average colonist saturation is greater than 5 it adds 0.1 per level, and if it's less than 5 it subtracts 0.1 per level.
-- If the average colonist housing is greater than 1, it adds 0.1 per level.
-- If 1 [Guard](../../source/workers/guard) level exists for each worker level (a Guard level 8 can keep a level 2 worker and a level 6 worker happy, for example), 0.1 is added for each average level more and 0.1 is subtracted for each average level less.
+These basic factors benefit from partial completion. Citizens will be happier with 5 shanks of saturation than 1, with 1 Guard for every 4 Workers than 1 Guard for every 10, and enjoy a level 2 house even more than a level 1 house. Colonists also become more happy as these values increase further, such as from upgrading a house to level 4 or 5, or being completely full on saturation.
 
-Each time a citizen is killed by a player, the overall happiness will decrease by 9 but won't go lower than 1. Each time a citizen dies otherwise (from a mob, falling, getting burned, etc) it will decrease by 1 (and won't go lower than 1).
+Each time a citizen, other than a guard, dies, other citizens need to mourn the next day, and will be less happy for the next three days.  All citizens become slightly less happy for the next day when injured.
+
+Every night, colonists will also become less happy if sick, homeless, unemployed, or with nothing to do at their jobs, and will become increasingly unhappy the longer the problem persists. Injury will reduce colonist happiness, as will guards who are unable to sleep. Colonists will also become less happy if fellow colonists become sick, starving, homeless, or unemployed.
+
+Surviving a raid without losing any colonists provides a colony-wide happiness boost.
```

Comment:
```suggestion
Surviving a raid without losing any colonists provides a colony-wide happiness boost.
```
Does this include Guards dying? If no -> "without losing any citizens"

Reply:
raidManager.onLostCitizen() fires for all citizens, both guards and non-guards, and that's what controls the raidswithoutdeath bonus.  It's only death happiness modifier and need to mourn that skip guard deaths.